deploy/charts/cert-manager/README.template.md

```diff
@@ -106,6 +106,13 @@ The following table lists the configurable parameters of the cert-manager chart
 | `affinity` | Node affinity for pod assignment | `{}` |
 | `tolerations` | Node tolerations for pod assignment | `[]` |
 | `topologySpreadConstraints` | Topology spread constraints for pod assignment | `[]` |
+| `livenessProbe.enabled` | Enable or disable the liveness probe for the controller container in the controller Pod. See https://cert-manager.io/docs/installation/best-practice/ to learn about when you might want to enable this livenss probe. | `false` |
```

Comment:
Once this is merged, I will update this documentation page with details about when and how to enable the livenessProbe.

Reply:
It'd be good to also ask people to report back if they actually find this probe useful so we can decide if we want it enabled by default